### ISSUE TITLE
Toastnotification when an app requests root

### ISSUE THREAD

**Issue by 4mede** [bmax121/APatch#803]
### Is your feature request related to a problem?/你的请求是否与某个问题相关？

No.

### Describe the solution you'd like/描述你想要的解决方案

I think Magisk implements this well. Ideally, users would be able to choose notifications, toasts, or nothing like on Magisk. Even if it's not on by default, I think it would be better than not having the option at all. For security reasons and so that new users know that they have to go to the app to enable superuser for an app to use root, I think this would be helpful.

### Describe alternatives you've considered/描述您考虑过的备选方案

None.

**Comment by 4mede** [comment]
Yes, but it's helpful to know when an app is using root and when an app that doesn't have it requests it. What if an app is using root and you don't want it to? What if an app is using it in the background all the time and so you want to revoke its permission? You won't know if there's no toast. What if there's a bypass like [this](https://github.com/canyie/MagiskEoP) and an app gets root permission without you authorizing it?

**Comment by 4mede** [comment]
Yes, it's fixed, and I wasn't saying it applies to APatch. But a similar vulnerability could be found in APatch.

**Comment by NanashiTheNameless** [comment]
Personally how I'd do it is you go into apatch and enable su for the app which makes the app able to see the su binary (as it is now) but when the app goes to use it apatch prompts to allow or deny that usage with the ability to always allow/always deny for that app.
